pmg/models/soundcloud.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from pmg import db
+
+
+class SoundcloudTrack(db.Model):
+
+    __tablename__ = "soundcloud_track"
+
+    id = db.Column(db.Integer, primary_key=True)
+    created_at = db.Column(
+        db.DateTime(timezone=True),
+        nullable=False,
+        default=datetime.utcnow
+    )
+    modified_at = db.Column(
+        db.DateTime(timezone=True),
+        nullable=False,
+        default=datetime.utcnow
+    )
+    file_id = db.Column(
+        db.Integer,
+        db.ForeignKey('file.id', ondelete='CASCADE'),
+        nullable=False
+    )
```

Comment:
If you define a relationship then you can include a backref so that you could go `file.soundcloud_track`:

```
file = db.relationship('File', lazy='joined', backref='soundcloud_track')
```